src/translations/sk.json

```
@@ -0,0 +1,41 @@
+{
+  "log_in": "Prihlásiť sa",
+  "log_out": "Odhlásiť sa",
+  "logged_in_as": "Prihlásený ako",
+  "logged_in": "Prihlásený užívateľ",
+  "logging_in": "Prebieha prihlásenie",
+  "logging_out": "Prebieha odhlásenie",
+  "sign_up": "Zaregistrovať sa",
+  "signing_up": "Registrácia",
+  "forgot_password": "Zabudli ste heslo?",
+  "recover_password": "Obnoviť heslo",
+  "send_recovery_email": "Odoslať e-mail pre obnovenie",
+  "sending_recovery_email": "Odosielanie e-mailu pre obnovenie",
+  "never_mind": "Naspäť",
+  "update_password": "Aktualizovať heslo",
+  "updated_password": "Aktualizácia heslá",
+  "complete_your_signup": "Dokončite registráciu",
+  "site_url_title": "Nastavenie vývoja",
+  "site_url_link_text": "Vymazať URL localhost",
+  "site_url_message": "Zdá sa, že používate lokálny server. Povedzte nám adresu URL svojho Netlify servera.",
+  "site_url_label": "Zadajte URL svojho servera Netlify",
+  "site_url_placeholder": "URL vášho Netlify servera",
+  "site_url_submit": "Nastaviť adresu URL",
+  "message_confirm": "Na váš e-mail bol odoslaný odkaz na potvrdenie registrácie, pokračujte kliknutím na tento odkaz.",
```

Using your updated suggestion, @michalhudecek!

```suggestion
  "message_confirm": "Potvrďte registráciu kliknutím na odkaz v správe, ktorú sme Vám práve zaslali na váš email.",
```